### PR TITLE
Add openSUSE Tumbleweed instructions

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: running.md [hunk 1]
```diff
@@ -247,3 +247,24 @@ sudo systemctl enable --now cockpit.socket
 ```
 
 If the first command fails with "database file for ... does not exist", refresh/update your system with `sudo pacman -Syu` first.
+
+
+### openSUSE Tumbleweed
+{:#tumbleweed}
+
+[Cockpit](https://software.opensuse.org/package/cockpit) is included in [openSUSE Tumbleweed](https://www.opensuse.org/):
```
Comment (martinpitt) [inline running.md:255]: Perhaps link to https://software.opensuse.org/package/cockpit ? Then viewers can directly see current versions and such.

File: running.md [hunk 2]
```diff
@@ -247,3 +247,24 @@ sudo systemctl enable --now cockpit.socket
...
+2. Enable cockpit:
+```
+# systemctl enable --now cockpit.socket
+```
+3. Open the firewall if necessary:
+```
+# firewall-cmd --permanent --zone=public --add-service=cockpit
+# firewall-cmd --reload
+```
+
```
Comment (martinpitt) [inline running.md:270]: Please remove the trailing newline.

**Comment by martinpitt** [comment]
https://martinpitt.github.io/cockpit-project.github.io/running.html updated. Very nice, thanks @lnussel !

File: running.md (hunk 2 above)
Comment (lnussel) [reply running.md]: done

File: running.md (hunk 1 above)
Comment (lnussel) [reply running.md]: That's where it already links to :) Changed the second link to point to TW though

**Pull request merged**